tests/test_run_reference_vmm.py

```diff
@@ -93,33 +95,59 @@ def shutdown(vmm_process):
     # TimeoutExpired exception and fail the test.
     vmm_process.wait(timeout=3)
 
+def setup_stdout_nonblocking(vmm_process):
```

Comment:
nit: can you add 2 newlines between all functions? That's the formatting that  PEP 8 expects. We do not yet have an integration test with that though :(

Reply:
Okay sure.